src/Domain/Structure/Direction.hpp

```diff
@@ -181,6 +181,26 @@ bool operator!=(const Direction<VolumeDim>& lhs,
   return not(lhs == rhs);
 }
 
+template <size_t VolumeDim>
+bool operator<(const Direction<VolumeDim>& lhs,
```

Comment:
Maybe document (either dox or C++) what the comparison is and that it's arbitrary without some deep motivation behind it? I worry that in 2 years we'll look at this and ponder why it's comparing in this specific way :)